docs/howto/krustlet-on-eks.md

```diff
@@ -1,3 +1,173 @@
 # Running Krustlet on Amazon Elastic Kubernetes Service (EKS)
 
-TODO: https://github.com/deislabs/krustlet/issues/130
+Currently, [EKS does not support](https://github.com/aws/containers-roadmap/issues/741) running managed node groups with custom Amazon Machine Images (AMI).
+
+However, it does appear the feature might be coming soon.
+
+Until that time, we can use [eksctl](https://eksctl.io/) to create and manage a node group with a custom Krustlet-based AMI.
+
+# Prerequisites
+
+The following tools are needed to complete this walkthrough:
+
+* [Amazon CLI](https://aws.amazon.com/cli/) - *use `aws configure` to set your access keys and default region*
+* [Packer](https://packer.io/)
+* [eksctl](https://eksctl.io/) 
+
+# Building the Krustlet-based AMI
+
+We will be using [Packer](https://packer.io/) to spin up an EC2 instance to build the AMI.
+
+There is a Makefile in `docs/howto/assets/eks` that will run packer for you.  It will use a `c5.2xlarge` EC2 instance to build the AMI with.  Use the `instance_type` variable to `make` to change the type of the EC2 instance used.
```

Comment:
Nit: write packer as `packer` to refer to the cli tool

Reply:
I'll fix.